velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -75,6 +75,12 @@ class ArrowDataBufferSink : public arrow::io::OutputStream {
     return sink_->isClosed();
   }
 
+  void abort() {
+    sink_->close();
```

Comment:
Not related to this PR, but we ideally need an abort for the `sink_` as well. I am guessing `close()` will write a partial file for LocalFileSink. Can you verify?

Reply:
Yes, `LocalFileSink::close()` calls the POSIX close, any buffered data will be flushed to the file before the file descriptor is closed.

Reply:
I was wondering if we should abort writing the file to disk instead of closing it and ending up with a file on disk. I guess aborting a file in progress is more involved based on the file system. `close`is fine for now.

Reply:
I see, like the `HadoopMapReduceCommitProtocol::abortTask` in Spark, which would clean up the files created by abort tasks.